operator/apis/machinelearning.seldon.io/v1/seldondeployment_webhook_test.go

```diff
@@ -1419,3 +1419,56 @@ func TestValidateShadowTraffic(t *testing.T) {
 	err = spec.ValidateSeldonDeployment()
 	g.Expect(err).ToNot(BeNil())
 }
+
+func TestValidateTwoPrepackTwoPods(t *testing.T) {
+	g := NewGomegaWithT(t)
+	err := setupTestConfigMap()
+	g.Expect(err).To(BeNil())
+	impl := PredictiveUnitImplementation(constants.PrePackedServerTensorflow)
+	spec := &SeldonDeploymentSpec{
+		Predictors: []PredictorSpec{
+			{
+				Name: "p1",
+				ComponentSpecs: []*SeldonPodSpec{
+					{
+						Spec: v1.PodSpec{
+							Containers: []v1.Container{
+								{
+									Image: "seldonio/mock_classifier:1.0",
+									Name:  "classifier",
+								},
+							},
+						},
+					},
+					{
+						Spec: v1.PodSpec{
+							Containers: []v1.Container{
+								{
+									Name: "classifier2",
+								},
+							},
+						},
+					},
+				},
+				Graph: PredictiveUnit{
+					Name: "classifier",
+					Children: []PredictiveUnit{
+						{
+							Implementation: &impl,
+							Name:           "classifier2",
+						},
+					},
+				},
+			},
+		},
+	}
+
+	name := "mydep"
+	namespace := "default"
+	spec.DefaultSeldonDeployment(name, namespace)
+	err = spec.ValidateSeldonDeployment()
+	g.Expect(err).ToNot(BeNil())
```

Comment:
Shouldn't the expectation be `.To(BeNil())` instead?

Reply:
good catch - fixed